Utkarsh Somvanshi/PersonalPortfolio/styles.css

```diff
@@ -0,0 +1,223 @@
+@import url('https://fonts.googleapis.com/css2?family=Kanit:wght@300&family=Raleway:wght@100;300;500&display=swap');
+
+* {
+    scroll-behavior: smooth;
+    margin: 0px;
+    font-family: 'Kanit', sans-serif;
+    box-sizing: border-box;
+}
+section {
+    background-color: yellow;
+    border: 1px solid black;
+}
+.first {
+    background-color: rgb(5, 5, 44);
+    height: 100vh;
+    color: white;
+    font-size: 30px;
+}
+ul {
+    display: flex;
+}
+ul li {
+    list-style: none;
+    width: 130px;
+    text-decoration: none;
+    padding: 0px 10px 0px 10px;
```

Comment:
Look out for the text margins and padding. They are not aligned properly. They don't have an equal amount of space between them.